SteamKit2 does not have a strong name

Which is odd because it does have a snk:
https://github.com/SteamRE/SteamKit/blob/80f48437d9955a9323df4244d22a3dfd4961af7f/SteamKit2/SteamKit2/SteamKit2.csproj#L5

> CSC : warning CS8002: Referenced assembly 'SteamKit2, Version=2.3.0.1249, Culture=neutral, PublicKeyToken=null' does not have a strong name.

Maybe it's due to the path that goes up the directory?

--- a/SteamKit2/SteamKit2/Properties/AssemblyInfo.cs
+++ b/SteamKit2/SteamKit2/Properties/AssemblyInfo.cs
@@ -30,5 +30,5 @@ using System.Runtime.InteropServices;
 [assembly: Guid( "cd42d0bc-72e4-451e-bcd0-5d09c4bca2a9" )]
 
 #if DEBUG
-[assembly: InternalsVisibleTo( "Tests" )]
+[assembly: InternalsVisibleTo( "Tests, PublicKey=00240000048000009400000006020000002400005253413100040000010001009330cecb41df66973ee75ded87679584c9ce29bebc7f6c6e1f7accb040a46c0ee0f43afee41b74619d5ef1fad2410c50a915537cc54be6ae89b902740feb402e7ae78d5a9570aa8086fafa814773cb7e9e0b010262154076ae6261051e4b8537d1165a5e9c35508844d5231b70d71d7a69a6f3056cd611cfeebd4fdba94bf5a6" )]
 #endif

--- a/SteamKit2/SteamKit2/SteamKit2.csproj
+++ b/SteamKit2/SteamKit2/SteamKit2.csproj
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <AssemblyOriginatorKeyFile>..\..\SteamKit.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
     <Description>.NET library that aims to interoperate with the Steam network.</Description>
     <PackageReleaseNotes>Release notes are available at https://github.com/SteamRE/SteamKit/releases/tag/2.4.0-Alpha.1</PackageReleaseNotes>
     <PackageIcon>steamkit_logo_128.png</PackageIcon>

--- a/SteamKit2/Tests/Tests.csproj
+++ b/SteamKit2/Tests/Tests.csproj
@@ -4,6 +4,8 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <RollForward>LatestMajor</RollForward>
     <ProjectGuid>{5BF6D076-399B-41CA-BAE7-37CE1C40CA67}</ProjectGuid>
+    <AssemblyOriginatorKeyFile>..\..\SteamKit.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
   </PropertyGroup>
 
   <ItemGroup>

